Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -418,6 +420,177 @@ bool Digitizer::convertHits(int det, const std::vector<o2::trd::HitType>& hits,
   return true;
 }
 
+bool Digitizer::convertSignals(const int det, int& arraySignal)
```

Comment:
I would suggest to rename this method to convertSignalsToDigits

Reply:
I agree, I wil rename it.